doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1207,6 +1207,13 @@ POSTed JSON example::
       "email": "pete@mailinator.com"
     }
 
+Delete an Authenticated User
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please read https://github.com/IQSS/dataverse/issues/1929 and make sure you understand the **consequences and limitations** before deleting a user account.
+
+``curl -X DELETE http://$SERVER/api/admin/authenticatedUsers/$identifier``
```

Comment:
It might be nice to give an example of $identifier. Is it `jsmith` or `@jsmith`?

Reply:
When I used it,  `jsmith` worked - I'll add that.